.github/workflows/compile-examples.yml

```diff
@@ -128,18 +126,16 @@ jobs:
               - examples/utility/Provisioning
           # Portenta
           - board:
-              type: mbed
+              type: mbed_portenta
             platforms: |
               # Install Arduino mbed-Enabled Boards via Boards Manager for the toolchain
```

Comment:
```suggestion
```

It is no longer being installed for the toolchain alone. The platform is now actually being used instead of being replaced by the one sourced from the Git repository.